src/javascripts/ng-admin/Main/component/service/config/Field.js

```diff
@@ -19,6 +19,7 @@ define(function (require) {
         order: null,
         identifier: false,
         format: 'yyyy-MM-dd',
+        parse: null,
```

Comment:
This can't be null, or it will break everything. The default value for `parse` should be `function(x) { return x; }`.